services/api-server/src/simcore_service_api_server/services_rpc/wb_api_server.py

```diff
@@ -0,0 +1,45 @@
+from dataclasses import dataclass
+from functools import partial
+
+from fastapi import FastAPI
+from fastapi_pagination import Page, create_page
+from servicelib.rabbitmq._client_rpc import RabbitMQRPCClient
+from servicelib.rabbitmq.rpc_interfaces.webserver.licenses.licensed_items import (
+    get_licensed_items as _get_licensed_items,
+)
+
+from ..exceptions.service_errors_utils import service_exception_mapper
+from ..models.pagination import PaginationParams
+from ..models.schemas.model_adapter import LicensedItemGet
+
+_exception_mapper = partial(service_exception_mapper, service_name="WebApiServer")
+
+
+@dataclass
+class WbApiRpcClient:
+    _rabbitmq_rpc_client: RabbitMQRPCClient
+
+    @_exception_mapper(rpc_exception_map={})
+    async def get_licensed_items(
+        self, product_name: str, page_params: PaginationParams
+    ) -> Page[LicensedItemGet]:
+        licensed_items_page = await _get_licensed_items(
+            rabbitmq_rpc_client=self._rabbitmq_rpc_client,
+            product_name=product_name,
+            offset=page_params.offset,
+            limit=page_params.limit,
+        )
+        return create_page(
+            [
+                LicensedItemGet.model_validate(elm.model_dump())
```

Comment:
We have to talk about this. I learned that we are doing it wrong getting so deep the schemas (i.e. `LicensedItemGet.`). The  service layer should know nothing about the schemas. 
Better to discuss offline. we need to take action and review the entire api-server on this... (I left a small example code with the idea above though ...)

Reply:
Yes, we discussed already the other day. But note here that `LicensedItemGet` here is the domain model. The api-server controls that one. So I am not coupled to the rpc interface through it. Of course I expose this one in the api-server's api. We could of course now have yet another model which is exposed in the api and convert into that one.